openxla-nvgpu/runtime/src/openxla/runtime/nvgpu/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 The IREE Authors
```

Comment:
It would be better to drive this from bazel_to_cmake (which will also aid downstream integration). Fine to come back and do that as a followup if not sure that it works (I had to make a number of tweaks to it).